File: C++/2021/1stOct_harshraj8843.cpp
#include <iostream>
using namespace std;

// prime number check function
bool isPrime(int n)
{
    if (n <= 1)
        return false;
    for (int i = 2; i < n; i++)
        if (n % i == 0)
            return false;
    return true;
}

int main()
{
    int n;
    cout << "Enter value of n : ";
    cin >> n;

    int sum1 = 0, sum2 = 0;
    int sum = sum1 + sum2;

    for (int i = 1; i <= n; i++)
    {
        if (i == 1)
        {
            sum1 = 1;
            cout << "1 ";
        }
        else
        {
            sum = sum1 + sum2;
            sum2 = sum1;
            sum1 = sum;
            if (i % 5 == 0 || isPrime(sum))
            {
                cout << "0 ";
            }
            else
            {
                cout << sum;
                cout << " ";
            }
        }
    }
    cout << "\n";
    return 0;
}

File: Python/2021/1stOct_harshraj8843.py
# value of n
n = int(input("Enter n : "))

# n-1 value
sum1 = 0
# n-2 value
sum2 = 0

sum = sum1 + sum2

# prime number check function
def isPrime(num):
    if num > 1:    
        for i in range(2, int(num/2)+1):
            if (num % i) == 0:
                return False
        else:
            return True
    else:
        return False

# Fibonacci Series print
for i in range(1, n+1):
    if(i == 1):
        sum1 = 1
        print('1', end=' ')
    else:
        sum = sum1 + sum2
        sum2 = sum1
        sum1 = sum
        if(i%5 == 0 or isPrime(sum)):
            print('0', end=' ')
        else:
            print(sum, end=' ')
print()
